web/src/components/HeaderLogo.tsx

```diff
@@ -49,8 +49,10 @@ const LogoContainer = styled.div`
 export const HeaderLogo = ({ link }: HeaderLogoProps): ReactElement => {
   const { viewportSmall } = useWindowDimensions()
   const { campaign, appName, icons } = buildConfig()
-  const currentDate = moment()
-  const showCampaignLogo = campaign && currentDate.isAfter(campaign.startDate) && currentDate.isBefore(campaign.endDate)
+  const currentDate = DateTime.now()
+  const campaignStartDate = DateTime.fromISO(campaign!.startDate)
+  const campaignEndDate = DateTime.fromISO(campaign!.endDate)
+  const showCampaignLogo = campaign && currentDate > campaignStartDate && currentDate < campaignEndDate
```

Comment:
You shouldn't use `!` if something might be undefined like in this case. Better revert it to the way it was before...
```suggestion
  const currentDate = DateTime.now()
  const showCampaignLogo = campaign && currentDate > DateTime.fromISO(campaign.startDate) && currentDate < DateTime.fromISO(campaign.endDate)
```